packages/management/addons-manager/upstream-metadata.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
```

Comment:
this file should be named as metadata.yaml

Reply:
renamed file name to metadata.yaml